lua/terrortown/menus/gamemode/commands/commands.lua

```diff
@@ -0,0 +1,176 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 99
+CLGAMEMODESUBMENU.title = "submenu_commands_commands_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
+    local plys = player.GetAll()
+    local plyChoices = {}
+
+    for i = 1, #plys do
+        local ply = plys[i]
+
+        plyChoices[i] = {
+            title = ply:Nick(),
+            value = ply,
+        }
+    end
+
+    -- RESTART ROUND --
+
+    local form = vgui.CreateTTT2Form(parent, "header_commands_round_restart")
+
+    form:MakeButton({
+        label = "label_execute_command",
+        buttonLabel = "label_button_round_restart",
+        OnClick = function(slf)
+            admin.RoundRestart()
+        end,
+    })
+
+    -- SLAY PLAYER --
+
+    local form2 = vgui.CreateTTT2Form(parent, "header_commands_player_slay")
+
+    local playerSlay = form2:MakeComboBox({
+        label = "label_player_select",
+        choices = plyChoices,
+        selectValue = plyChoices[1].value,
```

Comment:
Do we absolutely need to set a pre-selected value here?

I'd rather have an admin explicitly choose a player for these actions to prevent accidental mishaps.

Reply:
I absolutely see your point. Hm. I guess I'll change it and if nobody complains then it is fine

Reply:
The buttons `OnClick` needs a nil check then. Or can we disable the buttons outright if there is nothing selected in the dropdown?

Reply:
I could disable it, yes, Adds a bit of annoying code, but it should work

Reply:
![image](https://github.com/TTT-2/TTT2/assets/13639408/f001c6a5-9330-4d6e-bbd4-9d7d0ef4efac)

I extended our parenting system to support combo boxes as well. Right now it is pretty basic. No selection counts as false, any selection as true. But it works nice without much additional code

Reply:
neat!